platform/security/handler/security-handler-basic/src/test/java/org/codice/ddf/security/handler/basic/BasicAuthenticationHandlerTest.java

```diff
@@ -237,4 +253,20 @@ public void testExtractAuthenticationInfo() {
         result = (UPAuthenticationToken) handler.extractAuthenticationInfo(request);
         assertNull(result);
     }
+
+    @Test(expected = IllegalStateException.class)
+    public void testIllegalStateException() {
+        Parser parser = new XmlParser();
```

Comment:
Since you're executing a negative test for when the parser is null, you don't need this line here.